src/commands/synthetics/interfaces.ts

```diff
@@ -9,7 +9,7 @@ export interface MainReporter {
   log(log: string): void
   error(error: string): void
   initErrors(errors: string[]): void
-  testTrigger(test: Test, testId: string, executionRule: ExecutionRule, config: UserConfigOverride): void
+  testTrigger(test: Test, testId: string, executionRule: ExecutionRule, testOverrides: UserConfigOverride): void
```

Comment:
Since this MainReporter is in `interfaces.ts` it's publicly exported but as far as i understand even then changing the name here should still be backwards compatible 🤔